app/controllers/api/v1/referent_assignations_controller.rb

```diff
@@ -1,11 +1,27 @@
 class Api::V1::ReferentAssignationsController < Api::V1::AgentAuthBaseController
+  before_action :set_user, only: %i[create_many]
+
   def create
     referent_assignation = ReferentAssignation.new(referent_assignation_params)
     authorize(referent_assignation)
     referent_assignation.save!
     render_record referent_assignation
   end
 
+  def create_many
+    referent_assignations_params[:agent_ids].each do |agent_id|
+      referent_assignation =
+        ReferentAssignation.find_or_initialize_by(user: @user, agent_id: agent_id)
+      begin
+        authorize(referent_assignation)
+        referent_assignation.save
```

Comment:
```suggestion
        referent_assignation.save!
```
On veut que ça fail je pense si l'assignation ne se fait pas ?

Reply:
d'ailleurs on voudrait probablement s'économiser le save si l'assignation est déjà créée ?